src/DataGrid.tsx

```diff
@@ -931,10 +954,15 @@ function DataGrid<R, SR, K extends Key>(
   }
 
   function getLayoutCssVars() {
```

Comment:
Rather than returning a new object, we could do
```suggestion
  function getGridTemplateColumns() {
```

```jsx
style={{
  ...,
  gridTemplateColumns: getGridTemplateColumns()
}}
```